geth/transactions/txqueue_manager.go

```diff
@@ -125,8 +131,23 @@ func (m *Manager) WaitForTransaction(tx *common.QueuedTx) common.TransactionResu
 	}
 }
 
+// NotifyErrored sends a notification for the given transaction
+func (m *Manager) NotifyErrored(id common.QueuedTxID, inputError error) error {
```

Comment:
it would be better to move notifications to backend if possible, but this is also fine

Reply:
Ok, I'll add a new point to https://github.com/status-im/ideas/issues/63 to extract notifications to backend.